clinica/iotools/converters/oasis_to_bids/oasis_to_bids_utils.py

```diff
@@ -105,3 +104,33 @@ def write_sessions_tsv(bids_dir: Path, sessions_dict: dict) -> None:
             sep="\t",
             encoding="utf8",
         )
+
+
+def write_scans(
+    bids_dir: Path,
```

Comment:
I would put that on a single line since it is very short (unless the formatter did this by itself but it seems short enough to fit on a single line)